### DISCUSSION TITLE
Introductions

### DISCUSSION THREAD

**Discussion by jsmariegaard** [DHI/python-package-development#11] [General]
Hi 👋

My name is Jesper Sandvig Mariegaard, I work at DHI as Head of the Data Model Integration team. My GitHub name is @jsmariegaard  :-)  Looking forward to getting to know you all in the course

**Comment by ecomodeller** [comment]
Hi all,

I am Henrik Andersson, I work together with Jesper at DHI and I will be teaching this course together with Jesper.

You find me on GitHub as @ecomodeller.

**Comment by Shubbee** [comment]
Hi there! I'm Shubhneet Singh, a Coastal Engineer at DHI in Vancouver, Canada. You can find me on GitHub as @Shubbee. Big thanks to Jasper and Henrick for curating this course - I'm enthused to take my Py skills to the next level!

**Comment by RenanBRibeiro** [comment]
Hi everyone,
I am Renan Ribeiro, working as Hydrodynamic and Water Quality Modeler at DHI Americas, based in Santos-Brazil. I'm looking forward to starting the course and learning new skills. My GitHub name is @RenanBRibeiro

**Comment by dimsfarm** [comment]
Hello everyone
I am Christian Holm and I am a Data Integration Engineer at DHI. I am looking forward to improving my Python skills.

**Comment by connorchewning** [comment]
Hey all,

I am Connor Chewning and I work as a Remote Sensing Specialist at DHI. Looking forward to meeting everyone and improving our python skills together! My github is @connorchewning

**Comment by polSARfeatures** [comment]
Hi everyone!

I am Paul Connetable and I work as a remote sensing specialist at DHI. Thanks a lot for the course, I'm looking forward to it and learn lots of new exciting things :)

**Comment by ChrisGaszynski-DHI** [comment]
Hi Everyone!

I am Christopher Gaszynski and I am a Wastewater Engineer here at DHI. I work a lot with modelling of wastewater treatment plants in WEST, but I have recently started working with data-driven modelling in python. Looking forward to learning more and more about python :)

**Comment by tpilz** [comment]
Hi,

I'm Tobias Pilz, I joined DHI this year and work in the Berlin office. I'm (surface water) hydrologist and worked with different (surface) hydrological models with some focus on GIS and environmental statistics (mainly with R so far). Looking forward to expand my Python skills.

**Comment by pjgj-pjoynt** [comment]
Hi All! I'm Patrick Joynt, a Coastal Engineer at DHI in Ottawa, Canada. You can find me on GitHub as @pjgj-pjoynt.

**Comment by henriklocke** [comment]
Hi everyone. My name is Henrik Løcke and I am a water resources engineer located in Vancouver, Canada. I have used Python and VBA for a decade and look forward to take my skills to the next level with this course!

**Comment by FannyPicou** [comment]
Hello, my name is Fanny Picourlat. I work as a model engineer at Eau d'Azur. Eau d'Azur is the company in charge of drinking water production in Nice (France). We use DHI modeling tools to develop a decision support system for surface and groundwater management. With this course, I hope to develop my skills in Python in order to develop new modules for our system.

**Comment by rasa2k** [comment]
Hi everyone,
See you all tomorrow. @rasa2k

**Reply by jsmariegaard** [reply]
Hi @rasa2k , what is your real name 😊?

**Comment by johnkouman** [comment]
Hello, my name is Ioannis Koumaniotis. I am a master student at DTU and currently I am doing my master thesis with DHI. I had an introductory session in Github last semester and I found this opportunity to take my knowledge one step further. Also, I am looking forward to develop my skills in python. My name in github is @johnkouman.

**Comment by ovho-dhi** [comment]
Hi everyone,

I'm Oliver von Hoegen. I work at DHI in Berlin as a project engineer in water resources management and hydraulics. Great to be part of the course! Looking forward to expanding my python skills and working with you over the next few weeks! My name on github is [ovho-dhi](https://github.com/ovho-dhi)

**Comment by valerabaca** [comment]
Hi everyone! My name is Victor Valera, software engineer at Adamatics. Looking forward to boosting my python skills in the upcoming weeks 😁. On GitHub as @valerabaca

**Comment by wuwwen** [comment]
Hi everyone. My name is Wu Wen, a software engineer in M&E, T&I of DHI. I use C# for my daily developing. We launched a project of python recently. So, I would like to improve my python skills. Looking forward to seeing all of you. You can find me on GitHub @wuwwen

**Comment by wu-hao-liang** [comment]
Hi everyone. I am Wu Haoliang, a coastal engineer in DHI Singapore. We are planning to manage our modelling team's scripts using Github. Looking forward to learning from all of you. My github account is [wu-hao-liang](https://github.com/wu-hao-liang)

**Comment by prabhu-kalaivanan** [comment]
Hi All,
My name is Prabhu Kalaivanan, and I am a Senior DSS Expert in DHI, India. I liked the training and am excited to learn new Python skills. My GitHub account is **@prabhu-kalaivanan**.  I expect to get to know every one of you through the course.

**Comment by Digi-engi** [comment]
Hello all,
My name is Anders Hansen. I am resently graduated engineer within the water and environmental sector. I am currently a part of the digitalization team, which is why I want to participate in the course.

**Comment by jue-hu** [comment]
Hello everyone,
My name is Hu Jue, a software engineer in DHI, Shanghai. Looking forward to learning more Python skills from this training. My GitHub account is @jue-hu.

**Comment by buchmann-dhi** [comment]
Hello all,
I'm Bjarne, working with operational forecasting (Water Forecast / DHI BlueCast). 
I hope to learn some more about methods for code maintenance and distribution. I have worked with Python for quite some time.
My github tag is @buchmann-dhi

**Comment by Ratko-Proning** [comment]
Hello everyone,
My name is Ratko Ramuscak from Proning-DHI. I mainly work in hydrodynamic, hydrologic and flood forecast modeling. I look forward to learning and further developing my python skills through this course. My GitHub account is @Ratko-Proning

**Comment by woxh** [comment]
Hi everyone, I am Xin Huei, an Environmental Engineer in Environmental Solutions, DHI, previously in Singapore and now in Denmark. I am currently a master student at DTU and part-time with DHI. Looking forward to expand my Python skills. My GitHub user is @woxh

**Comment by ZhaoNianZH** [comment]
Hi everyone,
My name is Zhaonian Zhuang, and I work as a wastewater engineer at DHI China, based in Shanghai. I mainly work with the modeling of wastewater treatment plants using WEST and have some experience in data processing with Python. My GitHub account is @ZhaoNianZH.

**Comment by alberbm-dev** [comment]
Hello everyone! My name is Alberto Barragán-Moreno (here as @alberbm-dev) and I am a software developer at ADAMATICS, where I use my Python knowledge to develop a data analytics platform for citizen data scientists :smiley: Looking forward to this course!

**Comment by SaeedZee88** [comment]
Hi
My name is Saeed Zeinali (@SaeedZee88) and I am a modelling engineer at Jan De Nul. I am looking forward to using this course to improve my performance in my job.

**Comment by philliplapidez** [comment]
Hi! I am John Phillip Lapidez (@philliplapidez). I am a coastal engineer at DHI Singapore.

**Comment by ivercruz2590** [comment]
Hello My name is Iver Cruz (@ivercruz2590), I am a Geological Engineer in Eramet-Argentina.

**Comment by buchmann-dhi** [comment]
Hi Jesper (@jsmariegaard) and Henrik (@ecomodeller),
Just a clarification.
I read the present assignment as a git/function intro, but I just want to know if you expect "state of the art" coding style, eg with annotations as per PEP484 (https://peps.python.org/pep-0484/), or if we are using "plain ol' Py"), this would be the difference between eg:
```
def clean_spikes(data, max_jump=10):
    # Function implementation goes here
```
and
```
def clean_spikes(data: pd.Series, max_jump: int = 10) -> pd.Series:
    """ 
     Some nice descriptions
     Parameters: ...
     Returns: ...
    """
    # Function implementation goes here
```
I am assuming the former, because with the annotations would typically also go significantly more comments, and then we will (by far) exceed the 75 line suggested size.
Looking forward to a great coarse!
/Bjarne (@buchmann-dhi)

**Reply by ecomodeller** [reply]
We will get to docstrings (and a little bit about annotations / type hints) later in the course. Don't waste all your energy in the first exercise😉.

**Comment by thorosst** [comment]
Hello, I'm Thomas Rossetto and I work for DHI Italy. Officially, I'm involved in hydraulic modeling, but I use Python daily for data management and to interface with different 'worlds' (PostgreSQL, Mike+, web dashboard, ArcGIS, ...). I am very curious to see what I can learn from this course.

**Comment by V-RIOS** [comment]
Dears, good morning,

I'm Victor Ríos, I work for Eramine Sudamérica S.A (a Lithium mining enterprise, a subsidiary of Eramet group from France) located in Salta Province, Argentina. 
I'm a numerical flow modeler, I work with Feflow and am interested in learning and improving Python tools to optimize data processing steps, among other things
